src/components/post/PostTitle.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { TitleSection, TitleLabel, TitleInput } from './PostTitle.styled';
+
+interface PostTitleProps {
+    title: string;
+    onChange: (value: string) => void;
+}
+
+const PostTitle: React.FC<PostTitleProps> = ({ title, onChange }) => {
+    return (
+        <TitleSection>
+        <TitleLabel>제목</TitleLabel>
+        <TitleInput
+            placeholder="22자 이내로 간단하게 적어주세요"
+            value={title}
+            onChange={(e) => onChange(e.target.value)}
+            maxLength={22}
```

Comment:
```suggestion
            const MAX_LENGTH = 22;
            ...
            maxLength={MAX_LENGTH}
```

이런 식으로 최대 글자 수는 변수에 담아 사용하는 것도 유지보수 측면에서 좋을 것 같아요!